ci/README.md

```diff
@@ -0,0 +1,17 @@
+# CI - deploys via pyinvoke
+
+This folder contains code and YAML that drives the deployments of Kubernetes-based applications and services. For example,
+a deployment named `archiver` is configured in [the prod channel](./channels/prod.yaml) and is ultimatedly deployed
+by `invoke` (see below) calling `kubectl` commands.
+
+## invoke (aka pyinvoke)
+[invoke](https://docs.pyinvoke.org/en/stable/) is a Python framework for executing subprocess and building a CLI application.
```

Comment:
I'd go for a switch from `subprocess` to `subprocesses` here, but could go either way since the actual module being referred to is just called `subprocess`

Reply:
ah yeah that was meant to be plural

Reply:
done in https://github.com/cal-itp/data-infra/pull/2853